### PR TITLE
chore(i18n): Adding meta-tags.json, translations.json in German

### PR DESCRIPTION
**Pull request by stephanduesterhoeft**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Hi @stephanduesterhoeft, thanks for your patience and for all your hard work localizing the News UI.

I tested things locally and everything is off to a great start. Hope you don't mind, but I'll push a few other files to the `config/i18n/locales/german` directory to enable links to other parts of fCC. I also went ahead and fixed a few links and translation keys in `translation.json` so those things render properly.

Aside from that, I noticed a couple of minor things that I'll leave as review comments.

File: config/i18n/locales/german/translations.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "label": "Suche",
+    "accessible-name": "Sende deine Suchanfrage ab",
+    "placeholder": "Mehr als 8.000 Tutorials durchsuchen",
+    "see-results": "Zeige alle Ergebnisse für {{ searchQuery }}",
+    "no-tutorials": "Keine Tutorials gefunden"
+  },
+  "banner": {
+    "default": "Programmieren lernen - {{ <0> }}kostenloser 3.000-Stunden-Lehrplan{{ </0> }}",
+    "authenticated": "Unterstütze unsere gemeinnützige Organisation und unsere Mission. {{ <0> }}Spende an freeCodeCamp.org{{ </0> }}.",
+    "authenticated-donor": "Vielen Dank, dass du das freeCodeCamp durch deine Spenden unterstützt{{ <0> }}."
```
Comment (scissorsneedfoodtoo) [inline config/i18n/locales/german/translations.json]: The English version of this string is the following:

```
"authenticated-donor": "Thank you for supporting freeCodeCamp through {{ <0> }}your donations{{ </0> }}."
```

But the German version doesn't have wrappers (`{{ <0> }} and {{ </0> }}`) around any text:

```
    "authenticated-donor": "Vielen Dank, dass du das freeCodeCamp durch deine Spenden unterstützt{{ <0> }}."
```

Is it possible to wrap some text in those like with the English version of the string?

File: config/i18n/locales/german/translations.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+  "buttons": {
+    "forum": "Forum",
+    "donate": "Spende",
+    "load-more-articles": "Weitere Artikel laden"
+  },
+  "search": {
+    "label": "Suche",
+    "accessible-name": "Sende deine Suchanfrage ab",
+    "placeholder": "Mehr als 8.000 Tutorials durchsuchen",
```
Comment (scissorsneedfoodtoo) [inline config/i18n/locales/german/translations.json]: This is one string that will be different across all the publications. The English string refers to the number of published articles.

Could we change this placeholder to be something more generic like "Search our tutorials" without a number of published articles?

Feel free to use whatever wording you think is best.

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]

File: config/i18n/locales/german/translations.json (hunk 1 above)
Comment (stephanduesterhoeft) [reply config/i18n/locales/german/translations.json]: Excuse me.

Here is a hopefully correct suggestion.

"authenticated-donor": "Vielen Dank dafür, dass du freeCodeCamp durch {{ <0> }}deine Spenden{{ </0> }} unterstützt."

Thanks for your help

File: config/i18n/locales/german/translations.json (hunk 2 above)
Comment (stephanduesterhoeft) [reply config/i18n/locales/german/translations.json]: Of course.

"placeholder": "Suche in unseren Tutorials",

**Comment by stephanduesterhoeft** [comment]
Is everything okay now?

File: config/i18n/locales/german/meta-tags.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "freeCodeCamp Programmiertutorials: Python, JavaScript, Git & mehr",
+  "keywords": "freeCodeCamp, Programmierung, Frontend, Programmierer, Artikel, reguläre Ausdrücke, Python, JavaScript, AWS, JSON, HTML, CSS, Bootstrap, React, Vue, Webpack",
+  "description": "Stöbere in Tausenden von Programmiertutorials, die von Experten geschrieben wurden. Lerne Webentwicklung, Data Science, DevOps, Software-Sicherheit und hol dir Tipps für deine Karriere als Entwickler."
+}
```
Comment (majestic-owl448) [inline config/i18n/locales/german/meta-tags.json]:
```suggestion
}

```

maybe here needs a new line character

**Review by majestic-owl448 — DISMISSED** [review]
LGTM 👍

**Review by Sboonny — APPROVED** [review]

**Comment by Sboonny** [comment]
@scissorsneedfoodtoo 👋, can you give this another look?

**Comment by scissorsneedfoodtoo** [comment]
Thanks for your quick updates @stephanduesterhoeft, and for your reviews @ieahleen and @Sboonny. I tested things locally and everything LGTM.

To get the tests to pass, I've opened up https://github.com/freeCodeCamp/news/pull/416 in place of this PR.

**Pull request closed without merging**